### DISCUSSION TITLE
Astro support for auth-helpers

### DISCUSSION THREAD

**Discussion by AceCodePt** [supabase/supabase#17003] [Feature Requests]
Hi!
I already implemented the Astro needed support including the examples.
The support is for both server and client.
Truth to be told it's almost identical to some of NextJS implementations except different types.

**Comment by podbasico** [comment]
I have been waiting for this 🙏

> Hi! I already implemented the Astro needed support including the examples. The support is for both server and client. Truth to be told it's almost identical to some of NextJS implementations except different types.

Can you share your impelementation?
